packages/common/src/services/audius-backend/AudiusBackend.ts

```diff
@@ -2277,10 +2277,9 @@ export const audiusBackend = ({
           const data = action.data
           if ('playlist_id' in data) {
             entityType = data.is_album ? Entity.Album : Entity.Playlist
-            const playlist_ids = Array.isArray(data.playlist_id)
-              ? data.playlist_id
-              : [data.playlist_id]
-            return playlist_ids.map((playlist_id) => decodeHashId(playlist_id))
+            return data.playlist_id.map((playlist_id) =>
```

Comment:
possible for this to be non array still?

Reply:
lol! yes you think i did more than i really did, its all indentation basically, since we are memozing after rather than as part of the def itself, happy to handle these callouts anyway though :)